openpype/hosts/maya/plugins/publish/collect_look.py

```diff
@@ -357,6 +357,19 @@ def collect(self, instance):
             for vray_node in vray_plugin_nodes:
                 history.extend(cmds.listHistory(vray_node))
 
+            # handling render attribute sets
+            render_set_types = [
+                "VRayDisplacement",
+            ]
+            render_sets = cmds.ls(look_sets, type=render_set_types)
+            if render_sets:
+                history.extend(
+                    cmds.listHistory(render_sets,
+                                     future=False,
+                                     pruneDagObjects=True)
+                    or []
+                )
```

Comment:
This part also fixed an issue that I found when publishing regular look with VRay Displacement node that uses texture file.

The resource of file texture node that connected to the displacement node is not being remapped during look extraction, because the file node was not being collected here.